### ISSUE TITLE
Updates to TCEC-everything-compact.zip

### ISSUE THREAD

**Issue by skiminki** [TCEC-Chess/tcecgames#9]
The TCEC-everything-compact.zip archive file has currently the following contents:

- compact/everything/TCEC-everything-compet-no-frc.pgn
- compact/everything/TCEC-everything-compet-frc.pgn
- compact/everything/TCEC-everything.pgn

It would probably be better to move the PGNs to top level in the zip for convenience.

Further, instead of having all games in TCEC-everything.pgn, we should probably rename it to TCEC-evenything-bonus-test.pgn and only store the bonus and test games in that PGN.

The rationale for TCEC-everything.pgn was that I originally wanted to have just a single big PGN that has everything. But then there is the problem that the end user would have to subtract the competition games to only get the test+bonus games. 

To summarize, the new target for contents would be:

- TCEC-everything-compet-no-frc.pgn
- TCEC-everything-compet-frc.pgn
- TCEC-everything-bonus-test.pgn
